Web Development/README.md

```diff
@@ -112,9 +112,91 @@ This category is a curated collection of valuable web development resources cont
+  </tr>
+      <tr>
+    <td><a href="https://flask.palletsprojects.com/en/3.0.x/">Flask</a></td>
+    <td>Flask is a lightweight Python web framework that offers flexibility and simplicity for building web applications. It provides essential features for URL routing, templating, and request handling, allowing developers to create web apps with minimal boilerplate code.</td>
+  </tr>
+</table>
+
+#### Other Componets
+
+<table width="100%">
+  <tr>
+    <th>Resource Name</th>
+    <th>Description</th>
+  </tr>
+    <tr>
+    <td><a href="https://www.tutorialspoint.com/restful/index.htm">RESTful APIs</a></td>
+    <td>REST (Representational State Transfer) is an architectural style for designing networked applications. RESTful APIs adhere to REST principles and use standard HTTP methods (GET, POST, PUT, DELETE) for communication, making them scalable, stateless, and interoperable.</td>
+  </tr>
+    <tr>
+    <td><a href="https://www.tutorialspoint.com/graphql/index.htm">GraphQL</a></td>
+    <td>GraphQL is a query language and runtime for APIs developed by Facebook. It enables clients to request exactly the data they need and nothing more, reducing over-fetching and under-fetching of data. GraphQL APIs provide a more efficient and flexible alternative to REST.</td>
+  </tr>
+    <tr>
+    <td><a href="https://www.tutorialspoint.com/oauth2.0/index.htm">OAuth 2.0</a></td>
+    <td>Authentication is the process of verifying the identity of users, while authorization determines the actions users are allowed to perform. Common authentication mechanisms include OAuth 2.0 for delegated access and JWT (JSON Web Tokens) for stateless authentication.</td>
+  </tr>
+    <tr>
+    <td><a href="https://www.docker.com/101-tutorial/">Docker</a></td>
+    <td>Containerization is a lightweight virtualization method that enables developers to package applications and dependencies into isolated containers. Docker is a popular containerization platform that simplifies the process of building, shipping, and running containers.</td>
+  </tr>
+    <tr>
+    <td><a href="https://www.jenkins.io/doc/tutorials/">Jenkins</a></td>
+    <td>CI/CD pipelines automate the process of building, testing, and deploying applications. Jenkins and Travis CI are popular CI/CD tools that facilitate continuous integration and continuous delivery, ensuring code changes are consistently integrated and deployed to production.</td>
+  </tr>
+    <tr>
+    <td><a href="https://aws.amazon.com/documentation-overview/">AWS Documentation</a></td>
+    <td>Cloud platforms such as AWS (Amazon Web Services) and GCP (Google Cloud Platform) offer scalable infrastructure and services for hosting, managing, and deploying applications. They provide a wide range of compute, storage, database, and networking services for developers.</td>
+  </tr>
+</table>
```

Comment:
Instead of seperating it into different table, just add it into the courses since I think tutorial and courses is a little bit the same thing, and as you can notice in the [Table Of Contents](https://github.com/jfmartinz/ResourceHub/blob/0afe0594b21831a7633fee761509e2006b177f51/Web%20Development/README.md#table-of-contents) I used Tutorial or Courses.